### DISCUSSION TITLE
State Timeline - Export Total Time in State

### DISCUSSION THREAD

**Discussion by kevdel** [grafana/grafana#45372] [Dashboards, Panels & Visualizations]
For a each row in a stateline panel provide a way to extract the total time per state per view

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
